### DISCUSSION TITLE
How to download full report from GitHub Action?

### DISCUSSION THREAD

**Discussion by vplagov** [JetBrains/Qodana#74] [Q&A] [answered]
I have configured the Qodana Scan GitHub action as follows - https://github.com/plagov/project-euler/blob/master/.github/workflows/code_quality.yml

How can I download the full Qodana report from the job to explore it locally?
Can't find an option to download it from this action run - https://github.com/plagov/project-euler/actions/runs/1869000245

**Comment by tiulpin** [comment, marked as answer]
Hi, @plagov!

Here's the report in your job – it's added as a workflow artifact, and you can download it by clicking it's name

![CleanShot 2022-02-19 at 19 27 43@2x](https://user-images.githubusercontent.com/13538286/154809547-cc4cb73b-36dc-4aa8-9026-f4beda4627d1.png)

**Reply by vplagov** [reply]
Hi, @tiulpin! 
Thanks for the help!
Do I understand correctly, that it should be possible to see the report locally by opening the `./results/report/index.html`? It shows this for me:
![image](https://user-images.githubusercontent.com/16601852/154810846-c2b5572e-64f3-4ae6-8efc-fd0ccf874c75.png)

**Reply by tiulpin** [reply]
Sorry, it's not possible to just open the index.html at the moment. Please see https://www.jetbrains.com/help/qodana/html-report.html
